tools/c7n_left/c7n_left/core.py

```diff
@@ -336,8 +336,20 @@ def as_dict(self):
         }
 
 
+class LeftValueFilter(ValueFilter):
+    def get_resource_value(self, k, i):
+        if k.startswith('tag:') and 'tags' in i:
+            tk = k.split(':', 1)[1]
+            r = (i.get('tags', {}) or {}).get(tk)
```

Comment:
nit: since we're adding `or {}`, the default value for `.get()` doesn't seem to add any value

```suggestion
            r = (i.get('tags') or {}).get(tk)
```